kubetest/main.go

```diff
@@ -168,7 +173,14 @@ func defineFlags() *options {
 	flag.BoolVar(&o.up, "up", false, "If true, start the e2e cluster. If cluster is already up, recreate it.")
 	flag.StringVar(&o.upgradeArgs, "upgrade_args", "", "If set, run upgrade tests before other tests")
 
-	flag.BoolVar(&verbose, "v", true, "If true, print all command output.")
+	// The "-v" flag was also used by glog, which is used by k8s.io/client-go. Duplicate flags cause panics.
+	// 1. Even if we could convince glog to change, they have too many consumers to ever do so.
+	// 2. The glog lib parses flags during init. It is impossible to dynamically rewrite the args before they're parsed by glog.
+	// 3. The glog lib takes an int value, so "-v false" is an error.
+	// 4. It's possible, but unlikely, we could convince k8s.io/client-go to use a logging shim, because a library shouldn't force a logging implementation. This would take a major version release for the lib.
+	//
+	// The most reasonable solution is to accept that we shouldn't have made a single-letter global, and rename all references to this variable.
+	flag.BoolVar(&verbose, "verbose-commands", true, "If true, print all command output.")
```

Comment:
purge different docs, at least https://github.com/kubernetes/test-infra/blob/master/kubetest/README.md and the other e2e.go doc (I cannot find that :-\)

I think we flipped -v default to true now, we probably can just drop the flag in docs?

Reply:
I removed it, because it defaults true.